src/Summoner/Project.hs

```diff
@@ -19,7 +19,7 @@ import System.Process (readProcess)
 import Summoner.Ansi (errorMessage, infoMessage, successMessage)
 import Summoner.Config (Config, ConfigP (..))
 import Summoner.Default (currentYear, defaultGHC)
-import Summoner.License (License (..), customizeLicense, githubLicenseQueryNames, licenseNames)
+import Summoner.License (License (..), customizeLicense, showLicense, githubLicenseQueryNames, parseLicense)
```

Comment:
These imports are not sorted. Could you please run `stylish-haskell` on the files that you've changed, and it will sort them for you.

Reply:
I moved to HIE and tried `brittany` formatter in vscode, but I couldn`t force it to work. So, `stylish-haskell` is back now.

Reply:
I use [this plugin](https://marketplace.visualstudio.com/items?itemName=vigoo.stylish-haskell) for vscode and it's working for me, you could give it a try.